subsys/net/lib/coap/coap.c

```diff
@@ -981,6 +1064,55 @@ int coap_get_option_int(const struct coap_packet *cpkt, uint16_t code)
 	return val;
 }
 
+int insert_option(struct coap_packet *cpkt, uint16_t code, const uint8_t *value, uint16_t len)
```

Comment:
This should be static as well?

Reply:
The function is declared at the top of the file as `static`.